src/main/resources/db/changelog/changes/05-create-roles-table.yaml

```diff
@@ -0,0 +1,20 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-role-table
+      author: anton4uk
+      changes:
+        - createTable:
+            tableName: roles
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: role
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
```

Comment:
Please, add empty lines to your yaml files.
Make sure this red icon disappears
<img width="347" alt="image" src="https://github.com/annton-shakur/online-book-store-project/assets/140437083/3069e15c-6406-4580-b0f3-539b62f3ad7c">